configs/next/packages/libnxz/sources

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# POWER NX zlib compliant library source package and build info
+# ==============================================================
+#
+
+ATSRC_PACKAGE_NAME="POWER NX zlib compliant library"
```

Comment:
What about `NX GZIP library` instead?
I don't think "zlib" and "compliant" should be part of the name.

Reply:
I'm ok with changing it. I was using the same name from the projects github page.